scalatest/src/main/scala/org/scalatest/Suite.scala

```diff
@@ -2170,6 +2147,30 @@ used for test events like succeeded/failed, etc.
     (Map[A, B]() /: (for (m <- ms; kv <- m) yield kv)) { (a, kv) =>
       a + (if (a.contains(kv._1)) kv._1 -> f(a(kv._1), kv._2) else kv)
     }
+
+  // SKIP-SCALATESTJS-START
+  // Leave this around for a while so can print out a warning if we find testXXX methods.
+  def yeOldeTestNames(theSuite: Suite): Set[String] = {
```

Comment:
I think we can get rid of this now. It has been "a while" since the 2.0 release where we stopped supporting test methods on Suite.